app/controllers/system/admin/instance/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class System::Admin::Instance::UsersController < System::Admin::Instance::Controller
+  load_and_authorize_resource :instance_user, class: InstanceUser.name,
```

Comment:
Then it's not properly namespaced.

I think this controller is similar to `Course::UsersController`?  In `Course::UsersController` we are also fetching `@course_user` and present `@course_user.user`